src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -1260,6 +1253,24 @@ impl<'a> TypeChecker<'a> {
         .set_span(expr.span()))
     }
 
+    #[inline]
+    fn resolve_constant_literal(
```

Comment:
The misuse of `resolve_literal` instead of `resolve_constant_literal` could easily occur. I suggest removing `Literal::CurrentTimestmap` but instead parsing `CURRENT_TIMESTAMP` as a function call.

Reply:
Good advice, I'll adress later.